auslib/blobs/base.py

```diff
@@ -55,6 +55,41 @@ def createBlob(data):
     return blob_map[schema_version](**data)
 
 
+def merge_dicts(ancestor, left, right):
+    result = {}
+    for key in ancestor.keys() + left.keys() + right.keys():
```

Comment:
IIRC, keys() returns a list, so you may end up handling the same key twice. Maybe put this into a set?

Reply:
Good idea.